MAUI/DataGrid/unbound-row.md

```diff
@@ -229,4 +220,14 @@ private void DataGrid_QueryRowHeight(object? sender, DataGridQueryRowHeightEvent
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="Changing unbound row height" src="Images\unboundRow\maui-datagrid-unboundRowHeight.png" width="404"/>
+<img alt="Changing unbound row height" src="Images\unboundRow\maui-datagrid-unbound-row-height.png" width="404"/>
+
+## Get unbound rows
+
+You can get the unbound row at the specified row index using [GetUnboundRow]() method.
```

Comment:
The unbound row can be obtained by using the `GetUnboundRowAtRowIndex` method.

Reply:
Added